### PR TITLE
Reactified Statistics View

### PR DESCRIPTION
**Pull request by hotzenklotz**
### Mailable description of changes:
- improved rendering / loading indicators for Statistics --> Weekly view

### Steps to test:
- Smoke test

### Issues:
- part of #2009

------
- [x] Ready for review

### CODE REVIEW

File: app/assets/javascripts/admin/views/statistic/statistic_view.js [hunk 1]
```diff
@@ -1,57 +1,213 @@
...
+
+type State = {
+  achievements: {
+    numberOfUsers: number,
+    numberOfDatasets: number,
+    numberOfAnnotations: number,
+    numberOfTrees: number,
+    numberOfOpenAssignments: number,
+  },
+  timeEntries: Array<*>,
```
Comment (philippotto) [inline app/assets/javascripts/admin/views/statistic/statistic_view.js]: Don't we have a more precise here?

File: app/assets/javascripts/admin/views/statistic/statistic_view.js [hunk 2]
```diff
@@ -1,57 +1,213 @@
...
+                    key="user"
+                    render={user => `${user.firstName} ${user.lastName} (${user.email})`}
+                  />
+                  <Column
+                    title="Duration"
+                    dataIndex="tracingTimes"
+                    key="tracingTimes"
+                    render={tracingTimes =>
+                      tracingTimes.map(timeEntry => {
+                        const minutes = timeEntry.tracingTime / 1000 / 60;
```
Comment (philippotto) [inline app/assets/javascripts/admin/views/statistic/statistic_view.js]: Doesn't Momentjs offer some human readable utility function?

**Comment by philippotto** [comment]
Dropping the (data) base resolved the issue for me.

File: app/assets/javascripts/admin/views/statistic/statistic_view.js (hunk 2 above)
Comment (hotzenklotz) [reply app/assets/javascripts/admin/views/statistic/statistic_view.js]: Unfortunately there is no good way in moment.js. The time entry is a duration which requires `moment.duration`. However, durations don't support formatiing :-( There is a bunch of hacky solution out there, but at that point we can just do the simple math.

I have been following this thread for a year now without any progress:                         https://github.com/moment/moment/issues/1048

**Pull request merged**